modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -599,8 +598,7 @@ class Form extends Component {
 			this.refs.ruleBuilder.showRuleCreation();
 
 			this.hideAddButton();
-		}
-		else {
+		} else {
```

Comment:
Just out of curiosity did we change to this pattern again or was that just an SF error?

Reply:
SF error, I launched "VS Code prettier" after "gradlew sf"

Reply:
🙂 np

Reply:
> SF error, I launched "VS Code prettier" after "gradlew sf"

@eduardoallegrini Pro-Tip™: we have instructions [here](https://github.com/liferay/liferay-npm-tools/blob/master/packages/liferay-npm-scripts/README.md#visual-studio-code) for setting up VS Code to format The Liferay Way™ when you're in a liferay-portal checkout (ie. it doesn't mess with how Prettier works in other projects).